src/components/Dialog/index.tsx

```diff
@@ -210,7 +210,12 @@ const BodyWrapper = styled.div`
   }
 `
 
-const Body = styled.div`
+interface BodyStyles {
+  height?: string
```

Comment:
It probably makes more sense to pass a number as height, e.g. `375` instead of `375px`

Reply:
Hah yeah I dont have strong preference, I thought it'd be preferred to go the way to stay closer to the actual CSS (so including 'px'). But I'm cool with both.